Update commonobject.class.php

defrance opened the pull request:
Hide the margintable if click at the top of it (usefull when customer are just in front of you), reappear when you click again

defrance committed:
Update commonobject.class.php

Hide the margintable if click at the top of it (usefull when customer are just in front of you), reappear when you click again

defrance committed:
Update commonobject.class.php

defrance committed:
Update objectline_view.tpl.php

defrance committed:
Update commonobject.class.php

defrance commented:
added a little img to show where to click for hide/show

eldy commented:
Because feature does not look to me to be a massively used function, can you add a hidden constant where the area/image to click on is output. I prefer not having it (not really a standard behaviour) by default.

defrance commented:
yes good idea : if constant  = 1 it's open by default, if constant = 2 it's close by default
1 work on it

defrance committed:
Using hidden constant to use the feature

MARGININFO_HIDE_SHOW = 0 feature not used
MARGININFO_HIDE_SHOW = 1 feature used, displayed on open
MARGININFO_HIDE_SHOW = 2 feature used, not displayed on open
